backend/app.py

```diff
@@ -154,6 +153,10 @@ def create_walk_time_table():
             )
             selected_way_points = douglas_peucker_transformer.transform(path)
 
+            # we don't use the auto waypoints if the user has disabled them
+            if not options["auto_waypoints"]:
```

Comment:
Code makes sense. However, if auto_waypoints is false, we should not compute the douglas-peucker transform (to save some time). So I guess it makes sense to move the other code in the if/else statement.

Reply:
my fix does not work... see failed run